tools/bundler/index.js

```diff
@@ -1,27 +1,32 @@
 const path = require('path');
 const fs = require('fs');
 const traverse = require('json-schema-traverse');
+const { url } = require('inspector');
```

Comment:
@derberg for example this
This wasn't in your pr but was added in `next-major-spec`

Reply:
I don't see it being used in code

Reply:
Yes 
When you raised a pr on my branch then it was there but after that it was removed from master